lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -78,26 +89,31 @@ export default function GroupConfigSelector({
         })
       );
       setGroupSets(groupSets);
-    } catch (err) {
-      setFetchError(err);
+    } catch (error) {
+      setFetchError(error);
```

Comment:
Personal preference for either `error` (preferable) or if we must abbreviate, `e`, but I don't like `err` :P